test/plug/parsers_test.exs

```diff
@@ -179,6 +191,14 @@ defmodule Plug.ParsersTest do
     assert File.read!(file.path) == "hello\n\n"
     assert file.content_type == "text/plain"
     assert file.filename == "żółć.txt"
+
+    assert [
+             %{body: "{\"indisposed\": \"json\"}"},
+             %{body: "foo", headers: [{_, _}, {_, _}] = part_headers},
```

Comment:
We should assert on the headers!

Reply:
+1, broke out into individual assertions in latest